scrapy_crawlera/middleware.py

```diff
@@ -191,6 +208,14 @@ def process_exception(self, request, exception, spider):
             self._clear_dns_cache()
             self._set_custom_delay(request, self.connection_refused_delay)
 
+    def _retry_auth(self, response, request):
+        logging.debug("Retrying crawlera request for authentication issue")
+        retries = response.meta.get('crawlera_auth_retry_times', 0) + 1
+        retryreq = request.copy()
+        retryreq.meta['crawlera_auth_retry_times'] = retries
+        retryreq.dont_filter = True
```

Comment:
Maybe you could increment some kind of stats for auth issues similar to the RetryMiddleware
https://github.com/scrapy/scrapy/blob/master/scrapy/downloadermiddlewares/retry.py#L82